modules/caddyhttp/replacer.go

```diff
@@ -353,6 +354,9 @@ func getReqTLSReplacement(req *http.Request, key string) (interface{}, bool) {
 		case "client.certificate_pem":
 			block := pem.Block{Type: "CERTIFICATE", Bytes: cert.Raw}
 			return pem.EncodeToMemory(&block), true
+		case "client.certificate_pem_encoded":
+			block := pem.Block{Type: "CERTIFICATE", Bytes: cert.Raw}
+			return base64.StdEncoding.EncodeToString(pem.EncodeToMemory(&block)), true
```

Comment:
This is essentially base64-encoding a base64-encoding of the DER bytes and wrapping it in PEM blocks. I think all we need is the base64-encoded block, without PEM-encoding.